File: problemas/1000/1044.dart
import 'dart:io';

void main() {
  final entrada = stdin.readLineSync().split(' ');
  final a = int.parse(entrada[0]);
  final b = int.parse(entrada[1]);

  stdout.writeln(a % b == 0 || b % a == 0 ? 'Sao Multiplos' : 'Nao sao Multiplos');
}

File: problemas/1000/1044.js
const [a, b] = require('fs')
  .readFileSync('/dev/stdin', 'utf-8')
  .split(' ')
  .map(x => Number(x));

console.log(a % b === 0 || b % a === 0 ? 'Sao Multiplos' : 'Nao sao Multiplos');